cli/fathom_web/commands/pick.py

```diff
@@ -13,14 +12,25 @@
           type=Path(exists=True, file_okay=False, writable=True, dir_okay=True, allow_dash=True))
 @argument('number', type=int)
 def main(from_dir, to_dir, number):
-    """Move a given number of random things from one directory to another.
+    """Move a given number of HTML files and any extracted resources from one directory to another.
 
     Ignore hidden files.
 
     This is useful for dividing a corpus into training, validation, and testing
     parts.
-
     """
-    files = [f for f in listdir(from_dir) if not f.startswith('.')]
+    # Make these strings into ``Path``s so they are easier to work with
+    from_dir = pathlib.Path(from_dir)
+    to_dir = pathlib.Path(to_dir)
+
+    files = [f for f in from_dir.glob('*.html')]
```

Comment:
I might have said `list(from_dir...)` just for brevity, since you aren't doing any transformation of the values. Opinions?

Reply:
Yea, this line is awful. Thank you for catching it. I've queued up a change to get rid of it and use `list(from_dir...)` directly in the call to `sample()`.

Reply:
Heh, I'd stop short of "awful". ;-)